ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -223,6 +226,40 @@ public void shouldDropConnector() {
         is("mock-connector"));
   }
 
+  @Test
+  public void shouldCreateSourceConnectorSuccessfully() {
+    // Given:
+    String connectorName = "source-mock-connector";
+    connectNames.add(connectorName);
+
+    // When:
+    final RestResponse<KsqlEntityList> response = ksqlRestClient
+        .makeKsqlRequest("CREATE SOURCE CONNECTOR `" + connectorName + "` "
+            + "WITH(\"connector.class\"='org.apache.kafka.connect.tools.MockSourceConnector');");
+
+    //Then
+    assertThat(response.isSuccessful(), is(true));
+    assertNotEquals(response.getResponse().get(0), instanceOf(WarningEntity.class));
```

Comment:
This check is redundant with the one on the line below. Let's just keep the one below. For completeness, we can also add checks that the size of `response.getResponse()` is 1, and that the `CreateConnectorEntity` contains the expected connector name.